sleap/gui/widgets/video.py

```diff
@@ -1732,6 +1733,211 @@ def updateEdge(self, node: QtNode):
             self.setLine(line)
 
 
+class VisibleBoundingBox(QtWidgets.QGraphicsRectItem):
+    """
+    QGraphicsRectItem for user instance bounding boxes.
+
+    This object defines a scalable bounding box that encases an instance and handles
+    the relevant scaling operations. It is instantiated when its respective QtInstance
+    object is instantiated.
+
+    When instantiated, it creates 4 boxes, which are properties of the overall object,
+    on the corners of the overall bounding box. These corner boxes can be dragged to
+    scale the overall bounding box.
+
+    Args:
+        rect: The :class:`QRectF` object which defines the non-scalable bounding box.
+        parent: The :class:`QtInstance` to encompass.
+
+    """
+
+    def __init__(self, rect, parent=None):
```

Comment:
Some type hints here could help with clarity:

```suggestion
    def __init__(self, rect: QRectF, parent: Optional[QtInstance] = None):
```